client/containers/service/index.js

```diff
@@ -111,8 +109,8 @@ export default class Service extends Component {
    */
 
   getServiceTasks() {
-    const { tasks } = this.props.cluster;
-    const { serviceName } = this.props.service;
+    const {tasks} = this.props.cluster;
+    const {serviceName} = this.props.service;
     return tasks.filter(task => task.group === `service:${serviceName}`);
   }
-};
+};git 
```

Comment:
Looks like you accidently append `git` to this file. I'm guessing this will break things and needs to be removed.